charts/vsphere-cpi/v1.0.0/templates/vsphere-cloud-config-cm.yaml

```diff
@@ -6,10 +6,10 @@ metadata:
 data:
   vsphere.conf: |
     [Global]
-    secret-name = {{ .Values.vCenter.credentialsSecretName | quote }}
+    secret-name = {{ .Values.vCenter.credentialsSecret.name | quote }}
     secret-namespace = {{ .Release.Namespace | quote }}
-    port = {{ .Values.vCenter.port | default "443" | quote }}
-    insecure-flag = {{ .Values.vCenter.insecureflag | default "1" | quote }}
+    port = {{ .Values.vCenter.port | quote }}
+    insecure-flag = {{ .Values.vCenter.insecureFlag | quote }}
```

Comment:
I'm concerned about why the defaults were removed, but also it shouldn't matter.

Reply:
Not exactly removed, just moved to values.yaml and questions.yaml since that's the convention.